catalog.html

```diff
@@ -0,0 +1,253 @@
+                Мотель
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="product-group" value="apartments">
+                Апартаменты
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Стоимсоть, ₽:</legend>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <input class="price-input" type="text" name="price-min" placeholder="от" value="0">
+            </li>
+            <li class="catalog-filter-item">
+              <input class="price-input" type="text" name="price-max" placeholder="до" value="9000">
+            </li>
+          </ul>
+        </fieldset>
+        <button type="submit" name="submit">Применить</button>
+        <button type="reset" name="reset">Сбросить</button>
+      </form>
+    </section>
+
+    <section class="catalog-filter-results">
+      <h2 class="catalog-filter-results-title visually-hidden">Результаты поиска гостиниц</h2>
+      <span class="catalog-filter-results-count">Найдено гостиниц: 38</span>
+      <select class="select-control" name="sorting">
+        <option value="cheap">Сначала дешёвые</option>
+        <option value="expensive">Сначала дорогие</option>
+        <option value="popular">Сначала популярные</option>
+      </select>
+
+      <div class="btn-group">
+        <button class="btn-group-compact visually-hidden" type="button">Комактный вид</button>
+        <button class="btn-group-big visually-hidden" type="button">Большие фото</button>
+        <button class="btn-group-list visually-hidden" type="button">Списком</button>
+      </div>
+
+      <ul class="product-list">
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="#" alt="Вид на звездное небо через горы">
+            <h3 class="product-card-title">Amara Resort & Spa</h3>
+          </a>
+          <span class="product-card-type">Гостиница</span>
+          <span class="product-card-price">От 4000 ₽</span>
+          <a class="product-card-link" href="#">Подробнее</a>
+          <a class="favourites-link" href="#">В избранное</a>
+          <div class="rating">Рейтинг: 8,5</div>
+        </li>
+
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="#" alt="Горы на фоне тучи и радуги">
+            <h3 class="product-card-title">Villas at Poco Diablo</h3>
+          </a>
+          <span class="product-card-type">Гостиница</span>
+          <span class="product-card-price">От 5000 ₽</span>
+          <a class="product-card-link" href="#">Подробнее</a>
+          <a class="favourites-link" href="#">В избранное</a>
```

Comment:
```suggestion
          <a class="favourites-link" href="#">В избранном</a>
```
Так как в этом месте пример стилизации другой кнопки.